reflex/custom_components/custom_components.py

```diff
@@ -416,9 +416,8 @@ def _run_commands_in_subprocess(cmds: list[str]) -> bool:
 
 def _make_pyi_files():
     """Create pyi files for the custom component."""
-    from glob import glob
-
-    package_name = glob("custom_components/*.egg-info")[0].replace(".egg-info", "")
+    with open(CustomComponents.PYPROJECT_TOML, "rb") as f:
+        package_name = tomlkit.load(f).get("project", {})["name"]
```

Comment:
why the `.get` only for `"project"`... if that fails, we'll for sure get a `KeyError` looking up `"name"` in the empty dict.

Reply:
It was only to make the pyright happy. I'll refactor everything to a common function